### ISSUE TITLE
Does not reference `.browserslistrc` (had to rename to `browserlist`) + Still errors if no browsers defined

### ISSUE THREAD

**Issue by replete** [amilajack/eslint-plugin-compat#80]
This should lookup the filenames as described at the browserlist project.

# Repro

1. Download the `eslint-plugin-compat-demo` branch
2. Follow the instructions and run it successfully.
3. Remove the 'browserlist' line in `package.json`
4. Create a new file `.browserslistrc` and add the text 'IE9', run `npm lint again`:
``` bash
$ npm run lint

> eslint-plugin-compat-demo@1.0.0 lint /Users/phil/eslint-plugin-compat-demo
> eslint .


/Users/phil/eslint-plugin-compat-demo/App.js
  5:13  error  PaymentRequest is not supported in Samsung Browser 4, Safari 9.1, Opera 42, Opera Mini all, iOS Safari 9.3, IE Mobile 10, IE 10, Firefox 45, Edge 13, Chrome 49, Android Browser 4.4, Android UC Browser 11  compat/compat
  9:18  error  IntersectionObserver is not supported in Samsung Browser 4, Safari 9.1, Opera Mini all, iOS Safari 9.3, IE Mobile 10, IE 10, Firefox 45, Edge 13, Chrome 49, Android Browser 4.4, Android UC Browser 11      compat/compat

✖ 2 problems (2 errors, 0 warnings)
```

5. Rename `.browserslistrc` to `browserlist` and run `npm lint` again:

```bash
$ npm run lint

> eslint-plugin-compat-demo@1.0.0 lint /Users/phil/eslint-plugin-compat-demo
> eslint .


/Users/phil/eslint-plugin-compat-demo/App.js
  5:13  error  PaymentRequest is not supported in IE 9        compat/compat
  9:18  error  IntersectionObserver is not supported in IE 9  compat/compat

✖ 2 problems (2 errors, 0 warnings)
```

6. Delete the `browserlist` file entirely.
```bash
$ npm run lint

> eslint-plugin-compat-demo@1.0.0 lint /Users/phil/eslint-plugin-compat-demo
> eslint .


/Users/phil/eslint-plugin-compat-demo/App.js
  5:13  error  PaymentRequest is not supported in Samsung Browser 4, Safari 9.1, Opera 42, Opera Mini all, iOS Safari 9.3, IE Mobile 10, IE 10, Firefox 45, Edge 13, Chrome 49, Android Browser 4.4, Android UC Browser 11  compat/compat
  9:18  error  IntersectionObserver is not supported in Samsung Browser 4, Safari 9.1, Opera Mini all, iOS Safari 9.3, IE Mobile 10, IE 10, Firefox 45, Edge 13, Chrome 49, Android Browser 4.4, Android UC Browser 11      compat/compat
``` 

# Result

It seems that these behaviours are expected but not working:

* Should raise no eslint compat errors if there are no `browserlist` options specified (expected IMO)
* Should look for rules in the locations specified at https://github.com/ai/browserslist#queries

**Comment by amilajack** [comment]
Browserslist determines the default browsers. I'll look into the `.browserslistrc`

**Comment by ismay** [comment]
@amilajack The problem is that you're not using browserslist v2. Updating to that version will fix this.
